javascript/javascript1/week2/FlightBookingFunction.js

```diff
@@ -0,0 +1,34 @@
+function getFullname (firstName,surName,useFormalName,gender)
+{
+    if(!firstName || !surName || firstName === " " || surName === " ")
+    {
+        console.log('Provide valid name!');
+    }
+    else if(typeof(firstName) != "string" || typeof(surName) != "string")
+    {
+        console.log('Provide valid name!');
+    }
+    else if(useFormalName == true && gender === 'male')
```

Comment:
`useFormalName === true ` can be simplyfied just write `useFormalName`  and if you want to check for the false case you can use `!useFormalName`